src/containers/ruby-tab/sound-completer.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Define Ruby code completer for Motion Blocks
+ */
+class SoundCompleter {
+    getCompletions (editor, session, pos, prefix, callback) {
+        console.log("pos: ", pos);
+        const words = [
+            {
+                value: 'play_until_done("ニャー")',
+                meta: '終わるまでニャーの音を鳴らす' 
```

Comment:
この行の末尾に不要な空白があります。

```suggestion
                meta: '終わるまでニャーの音を鳴らす'
```

関連するエラー

```
  10:39  error  Trailing spaces not allowed                             no-trailing-spaces
```